kin-devplatform-sdk/src/main/java/kin/devplatform/marketplace/presenter/SpendDialogPresenter.java

```diff
@@ -130,7 +131,7 @@ private void submitAndSendTransaction() {
 			final String addressee = offer.getBlockchainData().getRecipientAddress();
 			final String orderID = openOrder.getId();
 
-			submitOrder(openOrder);
+			submitOrder(openOrder); // TODO: 13/01/2019 no callback here. need to check what is this case and maybe handle in case of not same blockchain versions
```

Comment:
we don't care about OOB spend as we don't have this use case

Reply:
ok, thanks